tensorflow_addons/image/transform_ops.py

```diff
@@ -255,7 +271,7 @@ def angles_to_projective_transforms(angles, image_height, image_width, name=None
 
 
 @tf.RegisterGradient("Addons>ImageProjectiveTransformV2")
-def _image_projective_transform_grad(op, grad):
+def _image_projective_transform_grad(op: TensorLike, grad: TensorLike) -> tf.Tensor:
```

Comment:
It's not needed to type the gradient functions. But if you really want to type it, op is a tensorflow operation (function), not a tensor.

Reply:
Thanks @gabrieldemarmiesse. I removed typing private function.